### DISCUSSION TITLE
from thinc.neural._classes.model import msgpack ImportError: cannot import name 'msgpack'

### DISCUSSION THREAD

**Discussion by KangChou** [explosion/spaCy#10815] [Help: Installation]
## Your Environment

* Operating System:ubuntu16.04
* Python Version Used:3.6.13
* spaCy Version Used:2.3.0
* Environment Information:

**Comment by polm** [comment]
Sorry you're having trouble with this. Could you share what your install command was (`pip install spacy`) and the whole output?

Also note that v2.3.0 is pretty old - it's been more than a year since we released v3, and I'd recommend giving it a try if you're using spacy for the first time.

**Reply by polm** [reply]
Also, since this came up in #10810 too - please don't just post an error message. Please explain what you were trying to do when the the error happened, and include a information like the command you used and the full stack trace.
